asset/js/ets_woo_qa_admin_script.js

```diff
@@ -51,4 +51,13 @@ jQuery(document).ready(function(){
 	        });
 	    }
 	});
+
+	jQuery('.ets_admin_apv').on('change', function(){
```

Comment:
Is this the admin approval checkbox of QA pairs, why do we need this JS to check/uncheck it?

Reply:
it use bcz only checkbox i am not solving that problem so i am using hidden input for showing check uncheck